docs/en/engines/table-engines/mergetree-family/annindexes.md

```diff
@@ -53,7 +60,7 @@ CREATE TABLE t
 (
   `id` Int64,
   `number` Tuple(Float32, Float32, Float32),
-  INDEX x number TYPE annoy GRANULARITY N
+  INDEX x number TYPE index_name(parameters) GRANULARITY N
```

Comment:
This is the generic syntax, right? Suggest to replace
- `x` by `ann_index_name`,
- `number` by `(number)` (or more generically: `(column1, column2, ...)` where l. 61 + l. 62 are changed to `column1` and `column2`, and
- `index_name` by `ann_index_type`.